open-company/open-company-web
# Dark mode fixes

Bug: right now the signup page has a dark line at the bottom.

Fix: do not set the dark mode class when user is visiting /login or /sign-up or any page under these

To test:
- remove your theme cookie (`localhost-ui-theme` or `staging-ui-theme` or `ui-theme`) so auto is defaulted
- set light theme in your system pref
- check that `/login` looks good
- check that `/sign-up` looks good
- check that `/sign-up/slack` looks good
- set dark theme in your system pref
- check that `/login` looks good
- check that `/sign-up` looks good
- check that `/sign-up/slack` looks good
- now log in and set your theme to light from the app menu (Auto button should be visible and cookie should have `light` as value)
- check that app pages such as AP, boards, panels etc are light
- logout
- check that `/login` looks good
- check that `/sign-up` looks good
- check that `/sign-up/slack` looks good
- now log in and set your theme to dark from the app menu (Auto button should be visible and cookie should have `light` as value)
- check that app pages such as AP, boards, panels etc are dark
- logout
- check that `/login` looks good
- check that `/sign-up` looks good
- check that `/sign-up/slack` looks good
